### DISCUSSION TITLE
All you need is links

### DISCUSSION THREAD

**Discussion by erlend-sh** [raffomania/linkblocks#42] [Ideas]
https://subconscious.substack.com/p/all-you-need-is-links

I think this article can serve as a core design tenet for Linkblocks. It's filled with guiding principles that align perfectly with the core 'game loop' of Linkblocks.

> ### You don’t need stars, hearts, upvotes, downvotes, you just need links
> 
> Social media, search, spam, recommendations — when faced with an abundance of information, we often find ourselves needing to separate wheat from the chaff. Many systems reach for stars, hearts, upvotes, and downvotes as quick fixes for user-generated quality signals. But what if we just used links?
> 
> ![image](https://github.com/raffomania/linkblocks/assets/583842/33664eda-feba-4ad6-a87a-4a3193108f48)
> 
> Inbound links can be used as a signal of quality. People are more likely to link to things that matter, and less likely to link to things that don’t. Sum up the backlinks to a page and you have a quality signal. This is the key insight behind [Google Pagerank](https://en.wikipedia.org/wiki/PageRank).

> ### You don’t need comments, you just need links
> 
> Comments are one of the core interaction primitives of today’s web. I can comment on your Facebook post, your Google Doc, your blog.
> 
> What is a comment? What is its structure? What is its function? It’s a bit of content that is conceptually related to a parent post. We could say that the comment points to the post, or something in the post. That’s a directional relationship.
> 
> Links also point to something. They describe a directional relationship. It’s not much of a leap to consider an inbound link a comment on the thing it links to. If we implement some form of [transclusion](https://en.wikipedia.org/wiki/Transclusion), we can express comments in terms of links. In fact, WordPress already does this with [Pingbacks](https://wordpress.org/support/article/trackbacks-and-pingbacks/).

[Curated collections](https://github.com/raffomania/linkblocks/discussions/37) for example is nothing more than a *list of links*.

To be clear though, the add-ons mentioned above (likes, comments etc.) can definitely be useful and should probably find their way into Linkblocks. However, they should all be solved for by pure linking first so as to push the link-graph to its limits.

**Comment by erlend-sh** [comment]
https://sheep.horse/2024/4/save_the_web_by_being_nice.html

**Reply by raffomania** [reply]
Hmm, what would be some good features for "being nice" on linkblocks? Something other than the traditional stars, boosts, reactions etc. 

Maybe taking a bookmark and posting it onto your own feed with a note saying why you like it.

**Reply by erlend-sh** [reply]
That’s another niceness for sure. The main takeaway from the article for me (which I should’ve included along with the link to begin with!) is that _sharing links is **nice**_.
